### ISSUE TITLE
Delete timeout when container has an active CIFS mountpoint

### ISSUE THREAD

**Issue by roosterfish** [lxc/lxc#4329]
# Required information

 * Distribution: Ubuntu
 * Distribution version: jammy
 * The output of
   * `lxc-start --version`: 5.0.0~git2209-g5a7b9ce67
   * `uname -a`: 5.19.0-46-generic

# Issue description

If a running privileged container gets destroyed whilst having an active CIFS mountpoint, running `lxc-destroy -n c1 -f` will hang for a while.

Originally posted here: https://github.com/canonical/lxd/issues/11421

# Steps to reproduce

Please see https://github.com/canonical/lxd/issues/11421#issuecomment-1640187863

# Information to attach

 - `dmesg`: `[22631.978096] CIFS: VFS: \\10.159.171.69 has not responded in 180 seconds. Reconnecting...`

**Comment by mihalicyn** [comment]
I'll take a look on that once time permits.

@stgraber can you assign me to this issue? It looks that I don't have a permission to do that by myself.

**Comment by fliiiix** [comment]
@mihalicyn any progress on this topic?

**Comment by mihalicyn** [comment]
> @mihalicyn any progress on this topic?

nope, sorry. But I have it in my TODO list. Most likely I will be able to investigate this issue on the next week.
